modules/parser/src/test/scala/org/dhallj/parser/DhallParserSuite.scala

```diff
@@ -30,4 +33,10 @@ class DhallParserSuite extends FunSuite() {
 
     assert(DhallParser.parse(""""# # # $ % ^ #"""") == expected)
   }
+
+  test("parse classpath import") {
+    val expected = Expr.makeClasspathImport(Paths.get("/foo/bar.dhall"), ImportMode.RAW_TEXT, null)
+
+    assert(DhallParser.parse("classpath:/foo/bar.dhall as Text") == expected)
```

Comment:
Minor question: what happens for `as Location` at the moment?

Reply:
Which should definitely be documented... Will update now!

Reply:
Sounds reasonable!